icu4c/source/tools/genren/genren.pl

```diff
@@ -154,7 +154,9 @@
             chop $qtype;
         } elsif ($mode =~ /Mach-O/) {
             ($address, $type, $_) = split(/ /);
-            if(/^_(.*)$/) {
+            if (/^(.*)\(/) {
+            	$_ = $1;
+            } elsif(/^_(.*)$/) {
```

Comment:
I think a comment would be nice. I think this does
// Remove a leading underscore.

If you have any idea for why this is done here, maybe you could add that... maybe even an example?

Reply:
Good idea.  Done.

As for why it does this, I don't know-- I can confirm that the `_` is there in the names of all the actual C functions in the `nm` output, even on Mac, but I don't know why it does that.

Reply:
Thanks for adding the comments, but please swap them :-)

Reply:
Dang it!  I caught that mistake, but then forgot to do a "git add".  Hopefully fixed now, and I also went ahead and (optimistically?) squashed my commits.